### DISCUSSION TITLE
Where is user organizations he belongs?

### DISCUSSION THREAD

**Discussion by 4FunAndProfit** [keycloak/keycloak#33937] [Q&A]
Hello,

I try to use the new keycloak organizations feature.

Yet, when i login with a user who belongs to 1 organization, i see in my jwt the current organization

Can't see it if multiple organizations nor the list of organizations

How to achieve to see the list of organizations ?

Thanks a lot lot in advance

**Comment by 4FunAndProfit** [comment]
Seems to be related to a bug, i wrote this issue: https://github.com/keycloak/keycloak/issues/33938

**Comment by fari-99** [comment]
if you have problem adding organizations to your jwt, i also have this problem. don't know if its the same problem, but i migrated from v24 to v26.0.0, i need to create new Client Scopes. i am using this docs.

https://github.com/p2-inc/keycloak-orgs/blob/main/docs/active-organization.md

please note, on the docs. it uses `Active Organization` but on the keycloak i can't find it, i use `Organization Membership`

hope this help

**Comment by 4FunAndProfit** [comment]
@fari-99 thanks for your help :) 

For information, they answer my issue also give a look for your issue too :)
